test/basics/hello_world.c

```diff
@@ -30,7 +30,7 @@ int main(int   argc,
     iprintf("%i shepherds...\n", qthread_num_shepherds());
     iprintf("  %i threads total\n", qthread_num_workers());
 
-    status = qthread_fork(greeter, (void*)123456789, &return_value);
+    status = qthread_fork(greeter, NULL, &return_value);
```

Comment:
Why this?

Reply:
That's just a readability fix. Sorry, for some reason I thought I'd included it with a different PR.